### PR TITLE
Update API docs and Module.md file in xmlutils module

### PR DESCRIPTION
**Pull request by ThisaruGuruge**

### CODE REVIEW

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/Module.md [hunk 1]
```diff
@@ -2,50 +2,3 @@
 
 This module provides utility functions to manipulate built-in `xml` data type.
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/Module.md:3]:
```suggestion
This module provides utility functions to manipulate the built-in `xml` data type. 
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/Module.md [hunk 2]
```diff
@@ -2,50 +2,3 @@
 
 This module provides utility functions to manipulate built-in `xml` data type. 
 It provides APIs to convert a `json` to an `xml`, or convert a `table` to an `xml`.
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/Module.md:4]:
```suggestion
It provides APIs to convert a `json` to an `xml` or convert a `table` to an `xml`.
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal [hunk 3]
```diff
@@ -16,39 +16,57 @@
...
+# ```ballerina
+# json data = {
+#     name: "John",
+#     age: 30
+# };
+# xml|error xmlValue = xmlutils:fromJSON(data);
+# ```
 #
-# + j - The json source
+# + jsonValue - The json source
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:38]:
```suggestion
# + jsonValue - The JSON source
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal [hunk 4]
```diff
@@ -16,39 +16,57 @@
...
 # + options - JsonOptions struct for JSON to XML conversion properties
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:39]:
```suggestion
# + options - The `JsonOptions` struct for JSON to XML conversion properties
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal [hunk 5]
```diff
@@ -16,39 +16,57 @@
...
-# + return - XML representation of the given JSON
-public function fromJSON(json? j, JsonOptions options = {}) returns xml|error {
-    return externFromJson(j, options);
+# + return - XML representation of the given JSON if successfully converted
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:40]:
```suggestion
# + return - XML representation of the given JSON if the JSON is 
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal [hunk 6]
```diff
@@ -16,39 +16,57 @@
...
+#            the JSON or else an `error`
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:41]:
```suggestion
#            successfully converted or else an `error`
```
Need to specify the error.

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal [hunk 7]
```diff
@@ -16,39 +16,57 @@
...
+public function fromJSON(json? jsonValue, JsonOptions options = {}) returns xml|error {
+    return externFromJson(jsonValue, options);
 }
 
 // This is a temporary fix until resolve the #19917
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:46]:
```suggestion
// This is a temporary fix until #19917 is resolved.
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal [hunk 8]
```diff
@@ -16,39 +16,57 @@
...
-function externFromJson(json? j, JsonOptions options = {}) returns xml|error = @java:Method {
+function externFromJson(json? jsonValue, JsonOptions options = {}) returns xml|error = @java:Method {
     name: "fromJSON",
     class: "org.ballerinalang.stdlib.xmlutils.ConvertUtils"
 } external;
 
 # Converts a table to its xml representation.
```
Comment (praneesha) [inline stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:52]:
```suggestion
# Converts a table to its XML representation.
```

File: stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal (hunk 7 above)
Comment (ThisaruGuruge) [reply stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:46]: This is a developer comment, not an API doc comment. Don't need to change them. ;)

Comment (praneesha) [reply stdlib/xmlutils/src/main/ballerina/src/xmlutils/xmlutils.bal:46]: Ack.

**Pull request merged**